ElementX/Sources/Services/Notification/Proxy/NotificationItemProxy.swift

```diff
@@ -39,8 +42,20 @@ protocol NotificationItemProxyProtocol {
     var isEncrypted: Bool { get }
 }
 
+extension NotificationItemProxyProtocol {
+    var identifier: String? {
```

Comment:
I wonder if both this and `notificationIdentifier` should simply become `id` and then we could use it for `Identifiable` in the future if we ever needed that.
```suggestion
    var id: String? {
```